README.md

```diff
@@ -98,7 +98,9 @@ LOGGING = {
             'formatter': 'logzioFormat',
             'token': '<<LOGZIO-TOKEN>>',
             'logs_drain_timeout': 5,
-            'url': 'https://<<LOGZIO-URL>>:8071'
+            'url': 'https://<<LOGZIO-URL>>:8071',
+            'retries_no': 4,
+            'retry_timeout': 2,
```

Comment:
Is there a place in the readme where we give a description for each parameter?

Reply:
Yes, adding :)